code/src/layouts/app-banner/messages/uk.json

```diff
@@ -1,4 +1,12 @@
 {
   "appBanner.header": "Неймовірні Ціни на Усі Ваші Улюблені товари",
-  "appBanner.paragraphfirst": "Ласкаво просимо до нашого магазину! Тут ви знайдете найкращі гаджети з усього світу та зможете обрати найкращий для вас товар."
+  "appBanner.paragraphfirst": "Ласкаво просимо до нашого магазину! Тут ви знайдете найкращі гаджети з усього світу та зможете обрати найкращий для вас товар.",
+  "subintro.firstParagraph": "Пікап з бордюру",
+  "subintro.secondParagraph": "Безкоштовна доставка замовлень на\n суму понад $50",
+  "subintro.thirdparagraph": "Низька ціна\n гарантована",
+  "subintro.fourthParagraph": "Доступний для\n вас 24/7",
+  "categorySection.title": "Купуйте за категорією",
+  "categorySection.computers": "Комп'ютери",
+  "categorySection.mobile": "Мобільний",
```

Comment:
You kept this translation inside own translation folder. #73 check this pr

Reply:
Done